R/utilities-checks.R

```diff
@@ -247,6 +247,20 @@ check_device = function(feature, action = "warn", op = NULL, maybe = FALSE,
     dev_name <- names(dev_cur)
   }
 
+  # {ragg} and {svglite} report capabilities, but need specific version
+  if (dev_name %in% c("agg_jpeg", "agg_ppm", "agg_png", "agg_tiff")) {
+    check_installed(
+      "ragg", version = "1.2.6",
+      reason = paste0("checking device support for ", feat_name, ".")
```

Comment:
```suggestion
      reason = paste0(for checking device support for ", feat_name, ".")
```

rlamg write pkg is required, so either to check or for checking

Reply:
Ah yes, good catch, thanks!